android/titanium/src/java/org/appcelerator/titanium/TiLaunchActivity.java

```diff
@@ -105,6 +113,9 @@ protected void loadScript()
 	@Override
 	protected void onCreate(Bundle savedInstanceState)
 	{
+		// set start time
+		TiApplication.START_TIME_MS = System.currentTimeMillis();
+
```

Comment:
We should **not** use the `System.currentTimeMillis()` method since it is affected by system clock changes. We should use the `SystemClock.uptimeMillis()` method instead, which is not affected by clock changes and only counts up. It's perfect for timing measurements and for implementing timeouts.
https://developer.android.com/reference/android/os/SystemClock
https://developer.android.com/reference/android/os/SystemClock#uptimeMillis()